.github/workflows/CI.yml

```diff
@@ -320,6 +320,8 @@ jobs:
         run: |
           # allow newer gcc
           sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+          # use a recent Mesa version
+          sudo add-apt-repository ppa:kisak/kisak-mesa -y
```

Comment:
I see the repo added, but what package is coming from here?

Reply:
Thanks for asking. I added a comment giving a few more details. Here's the complete list as extracted from the CI build logs:
```
libwayland-server0:amd64 (1.20.0-1ubuntu0.1~kisak~f)
libwayland-bin (1.20.0-1ubuntu0.1~kisak~f)
libva2:amd64 (2.14.0-1~kisak1~f)
libglapi-mesa:amd64 (24.0.5~kisak1~f)
libvdpau1:amd64 (1.4-3build2~kisak~f)
libllvm15:amd64 (1:15.0.7-10~kisak1~f)
libwayland-egl1:amd64 (1.20.0-1ubuntu0.1~kisak~f)
libvdpau-dev:amd64 (1.4-3build2~kisak~f)
libdrm-common (2.4.120-1~kisak~f)
libwayland-client0:amd64 (1.20.0-1ubuntu0.1~kisak~f)
libdrm2:amd64 (2.4.120-1~kisak~f)
libva-drm2:amd64 (2.14.0-1~kisak1~f)
libwayland-cursor0:amd64 (1.20.0-1ubuntu0.1~kisak~f)
libva-x11-2:amd64 (2.14.0-1~kisak1~f)
libva-glx2:amd64 (2.14.0-1~kisak1~f)
libdrm-amdgpu1:amd64 (2.4.120-1~kisak~f)
libdrm-nouveau2:amd64 (2.4.120-1~kisak~f)
libgbm1:amd64 (24.0.5~kisak1~f)
libdrm-radeon1:amd64 (2.4.120-1~kisak~f)
libdrm-intel1:amd64 (2.4.120-1~kisak~f)
libgl1-mesa-dri:amd64 (24.0.5~kisak1~f)
libwayland-dev:amd64 (1.20.0-1ubuntu0.1~kisak~f)
libgbm-dev:amd64 (24.0.5~kisak1~f)
libegl-mesa0:amd64 (24.0.5~kisak1~f)
libva-wayland2:amd64 (2.14.0-1~kisak1~f)
mesa-va-drivers:amd64 (24.0.5~kisak1~f)
mesa-vdpau-drivers:amd64 (24.0.5~kisak1~f)
libdrm-dev:amd64 (2.4.120-1~kisak~f)
libglx-mesa0:amd64 (24.0.5~kisak1~f)
libva-dev:amd64 (2.14.0-1~kisak1~f)
va-driver-all:amd64 (2.14.0-1~kisak1~f)
vdpau-driver-all:amd64 (1.4-3build2~kisak~f)
libgl1-mesa-dev:amd64 (24.0.5~kisak1~f)
libegl1-mesa-dev:amd64 (24.0.5~kisak1~f)
```
It's quite an update over 20.04's mesa; some testing can't hurt.